build.gradle

```diff
@@ -221,7 +223,7 @@ dependencies {
     // runtimeOnly fg.deobf("curse.maven:terralith-513688:3606078")
     compileOnly fg.deobf("curse.maven:diet-443570:4813890")
 
-    compileOnly fg.deobf("dev.gigaherz.jsonthings:JsonThings-${minecraft_version}:${json_things_version}")
+    compileOnly fg.deobf("curse.maven:json-things-541617:5593943")
```

Comment:
Why are you using curse maven? Json Things still has a proper maven on 1.20.1

Reply:
My guess is that it is because Json Things 0.9.2 (the most recent version) isn't available on the Maven repository, though that is only speculation on my part.  I just pushed a commit to switch back to using the Maven repo and use 0.9.1 instead!

Reply:
Yep, I couldn't find the newest version on their maven. But now we are using it again.

Reply:
Is there a reason you need the newest version?